atom/browser/api/atom_api_app.cc

```diff
@@ -1021,6 +1022,36 @@ std::vector<mate::Dictionary> App::GetAppMetrics(v8::Isolate* isolate) {
   return result;
 }
 
+static mate::Dictionary ConvertDictionary(
+    v8::Isolate* isolate, const base::DictionaryValue &dict) {
+  auto result = mate::Dictionary::CreateEmpty(isolate);
+
+  for (base::DictionaryValue::Iterator iterator(dict);
+       !iterator.IsAtEnd();
+       iterator.Advance()) {
+    auto& key = iterator.key();
+    auto& value = iterator.value();
+    if (value.is_string()) {
+      std::string strValue;
+      if (value.GetAsString(&strValue)) {
+        result.Set(key, strValue);
+      }
+    }
+  }
+
+  return result;
+}
+
+mate::Dictionary App::GetGPUFeatureStatus(v8::Isolate* isolate) {
+  auto result = mate::Dictionary::CreateEmpty(isolate);
+
+  if (auto status = content::GetFeatureStatus()) {
```

Comment:
It does not work. There does not seem to be a converter from `base::DictionaryValue`. In generic case the values can have different types. In this case it can only be a string.